resources/asciidoctor/lib/care_admonition/extension.rb

```diff
@@ -34,19 +34,48 @@ def activate(registry)
   # Block care admonition.
   class ChangeAdmonitionBlock < Asciidoctor::Extensions::BlockMacroProcessor
     use_dsl
-    name_positional_attributes :passtext
+    name_positional_attributes :passtext, :issue_url
 
     def initialize(role, default_text)
       super(nil)
       @role = role
       @default_text = default_text
     end
 
+    def generate_text(text, issue_url)
+      if text&.start_with?('http', '{issue}')
+        issue_url = text
+        text = @default_text
+      else
+        issue_url = issue_url
+        text ||= @default_text
+      end
+      text = add_issue_text(text, issue_url) if issue_url
+      text
+    end
+
+    def add_issue_text(text, issue_url)
+      issue_num = get_issue_num(issue_url)
+      issue_text = <<~TEXT
+        For feature status, see #{issue_url}[\##{issue_num}].
+      TEXT
+      text + ' ' + issue_text
+    end
+
+    def get_issue_num(issue_url)
+      if issue_url.start_with?('http')
+        issue_num = issue_url.split('/').last
+        issue_num.chomp!('/')
+      else
+        issue_num = issue_url.sub('{issue}', '')
+      end
+      issue_num
```

Comment:
I believe it is a little more idiomatic in ruby to do
```
    def get_issue_num(issue_url)
      if issue_url.start_with?('http')
        issue_url.split('/').last.issue_num.chomp('/')
      else
        issue_url.sub('{issue}', '')
      end
    end
```

Though I tend to prefer "early returns" from my time doing java:
```
  def get_issue_num(issue_url)
     return issue_url.split('/').last.chomp('/') if issue_url.start_with?('http')
     issue_url.sub('{issue}', '')
  end
```

Reply:
Thanks @nik9000. I'll go with the second option. Rubocop seems fine with it.